npm/src/proto_based.ts

```diff
@@ -264,7 +260,9 @@ function computeKvCheckMessage(
   return {
     key: packKey(key),
     versionstamp: (versionstamp === null || versionstamp === undefined)
-      ? emptyVersionstamp
+      /* in proto3 all fields are optional, but the generated types don't
+       * like it, so we have to cast it manually */
+      ? undefined as unknown as Uint8Array
```

Comment:
I think you can just use `new Uint8Array()` here - the zero value is not distinguishable from an unset value in protobuf.

Reply:
While it will work, it will be encoded and take up some bandwidth (I actually measure it output).